app/assets/stylesheets/_variables.scss

```diff
@@ -26,29 +21,14 @@ $dark-gray: #333;
 $medium-gray: #999;
 $light-gray: #DDD;
 
-// Background Color
-$base-background-color: white;
-
 // Font Colors
 $base-font-color: $dark-gray;
-$base-accent-color: $blue;
-
-// Link Colors
-$base-link-color: $base-accent-color;
-$hover-link-color: darken($base-accent-color, 15);
-$base-button-color: $base-link-color;
-$hover-button-color: $hover-link-color;
+$action-color: $blue;
```

Comment:
The thought here was to be clear about what color `$action-color` was. Off the top of my head I don't have a clear sense of what `#477dca` is. I could totally be convinced of deleting more things though.